src/components/reservation/OpenOrderResponse.tsx

```diff
@@ -0,0 +1,69 @@
+import FocusTrap from "focus-trap-react";
+import React from "react";
+import { useDispatch } from "react-redux";
+import { closeModal } from "../../core/modal.slice";
+import { useText } from "../../core/utils/text";
+import { Button } from "../Buttons/Button";
+import { OpenOrderMutation } from "../../core/dbc-gateway/generated/graphql";
+import { translateOpenOrderStatus } from "./helper";
+
+type OpenOrderResponseProps = {
+  title: string;
+  modalId: string;
+  openOrderResponse: OpenOrderMutation;
+};
+
+const OpenOrderResponse: React.FC<OpenOrderResponseProps> = ({
```

Comment:
I made a general modal message component that could be used here. I think we should look at it together...